### ISSUE TITLE
Normalize data for bibliographic references

### ISSUE THREAD

**Issue by wlpotter** [srophe/britishLibrary-data#44]
These are somewhat all over the place.

For ms references these should be:

```
<bibl>Add. x<ptr target="$URI"/></bibl>
```

Optional cited range

**Comment by wlpotter** [comment]
Is this a priority for first release?

**Comment by wlpotter** [comment]
Another aspect of this is implementing the manuscript URIs so we both link within the project using the BL URI and reference the Syriaca ms URI. See [this comment](https://github.com/srophe/britishLibrary-data/issues/21#issuecomment-1139710466) and discussion. We will follow the e-GEDSH and BQG practice and use space-separated URIs in the `ptr/@target` attribute. We'll put the BL URI first by convention.

**Comment by wlpotter** [comment]
Find all the bibl references with a BL ms (contain the string "Add"). And see which are well-formatted. We will wait on references to books or mss that aren't in Wright

**Comment by wlpotter** [comment]
Using the structures enumerated in the above quoted CSVs, I was able to divide the work into three different tasks:

- [x] about 70 bibls needed to be updated by hand. I went ahead and made those changes
- [ ] about 280 bibls needed to be matched to the URI based on the cited shelfmark. I've done this and am now working on the script to add these URIs to the corresponding bibls
- [ ] in addition, about 150 bibls were in a different format that could be machine-parsed. These will need an update script to parse the current reference, extract the URI and citedRange information, and then replace the bibl with an updated and normalized version. I'm working on the script to do this.

**Comment by wlpotter** [comment]
Two further notes with things to check on here:

- check empty `@unit` attributes on citedRanges. These are likely where Wright refers to an item's number rather than a locus, so we probably want to just remove these attributes
- check where we have listBibl elements. This was an old style of encoding that we may want to update. This probably needs a separate issue as the most important change is to normalize the bibls themselves.
- check on empty `ptr` elements. These are likely references to manuscripts not in Wright's catalogue.

**Comment by wlpotter** [comment]
Other issues have been made. This issue is fixed with #106 , at least for ms references (see #110 for a similar normalization of non-ms references in tei:bibls)
